### PR TITLE
 Extension: Update Translate (Chinese Simplified)

### PR DESCRIPTION
**Pull request by yangyangdaji**
Extension: Update Translate (Chinese Simplified)

### CODE REVIEW

File: web/packages/extension/assets/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
     "settings_show_swf_download": {
         "message": "在右键菜单中显示SWF下载选项"
     },
+    "settings_autostart": {
+        "message": "没有加载画面的自动播放 (点击内容取消静音)"
```
Comment (urain39) [inline web/packages/extension/assets/_locales/zh_CN/messages.json:15]: 没有 -> 不带
加载画面 -> 启动画面

这样好像更容易理解

Comment (yangyangdaji) [reply web/packages/extension/assets/_locales/zh_CN/messages.json]: OK

**Review by danielhjacobs — APPROVED** [review]
I don't actually know Chinese but according to Google Translate this seems to be a more accurate description of that setting.

**Review by urain39 — APPROVED** [review]

**Review by relrelb — APPROVED** [review]
Thank you!

**Comment by yangyangdaji** [comment]
OK, thank

**Pull request merged**